site/src/docs/components/header/index.mdx

```diff
@@ -45,6 +45,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 - Always position the Header.UniversalBar to the top of the page. Keep the bar in place so the user can always easily locate it. It is recommended to include the Header.UniversalBar on every page of the service.
 - It's recommended to pass an ariaLabel prop for the Header.UniversalBar to describe it for the screen readers, e.g. "Quick links".
 - The Header.UniversalBar component is built to follow <InternalLink href="/foundation/design-tokens/breakpoints">HDS breakpoint tokens</InternalLink>. On smaller screens (576px and down) the Header.UniversalBar will not be displayed on the page and its contents should be found inside the HeaderNavigationMenu component.
+- Header supports two themes: light (default) and dark. See example for light theme <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features">in Storybook</Link> and for dark theme <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features-dark-theme">in Storybook</Link>
```

Comment:
This example is simple so I think you can just add a new section for Theming in index and code pages. See [Navigation colour variations](--nav-link-font-color) for inspiration.

Reply:
Bump for this, or do you disagree? I think this would be alright to display in doc site as they are displayed in old Navigation docs too.